components/Card.js

```diff
@@ -6,16 +6,20 @@ export const CardElement = styled.article`
   text-align: center;
   border: 3px grey solid;
   border-radius: 10px;
+  width: 18rem;
+  p {
+    font-weight: ${(props) => (props.isBold ? "bold" : "normal")};
+  }
```

Comment:
I think rather than passing the `isBold` prop to the `CardElement` component and having a nested style inside of your "CardElement" component for your `p` type selector, I'd suggest creating some type of text component that is passed `isBold` instead.